conpot/protocols/bacnet/bacnet_server.py

```diff
@@ -0,0 +1,410 @@
+# Author: Peter Sooky <xsooky00@stud.fit.vubtr.cz>
```

Comment:
Could you include the gnu public license in the header(like we do on the other source files)?